backend/webserver/tests/test_views.py

```diff
@@ -2317,6 +2636,54 @@ def test_get_follow_request_received_from_a_team10_remote_sender(self):
         self.assertEqual(status.HTTP_200_OK, response.status_code)
         self.assertEqual(expected_output, response.data)
 
+    @responses.activate
+    def test_get_follow_request_received_from_a_team16_remote_sender(self):
```

Comment:
I had given you a similar feedback on your previous PR related to team 10's converter but we don't need this test because it doesn't test anything about the converter.